applications/algo_stablecoin_proposal.md

```diff
@@ -0,0 +1,190 @@
+# W3F Grant Proposal
+
+> This document will be part of the terms and conditions of your agreement and therefore needs to contain all the required information about the project. Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with a `>` (such as this one) can be removed.
+>
+> See the [Grants Program Process](https://github.com/w3f/Grants-Program/#pencil-process) on how to submit a proposal.
+
+- **Project Name:** Algorithmic Stablecoin Design and Proof of Concept
+- **Team Name:** Stardust Labs Inc.
+- **Payment Address:** 0xF3d5F194D9eF961a85a4d5be05EFda7B2B33005d (DAI, Ethereum Mainet)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+> ⚠️ _The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe._
+
+## Project Overview :page_facing_up:
+
+This proposal is in response to an RFP for an Algorithmic Uncollateralized Stablecoin (Specifically: https://github.com/w3f/Grants-Program/blob/master/rfps/uncollateralized-stablecoin.md).
```

Comment:
```suggestion
This proposal is in response to an RFP for an Algorithmic Uncollateralized Stablecoin Research (Specifically: https://github.com/w3f/Grants-Program/blob/master/rfps/under_development/uncollateralized-stablecoin-research.md).
```

Reply:
Note: We recently restructured the RFP directory which is why this link is outdated.

Reply:
@adit313 feel free to accept my fix :D